core/src/main/java/bisq/core/payment/UpholdAccount.java

```diff
@@ -44,4 +44,13 @@ public void setAccountId(String accountId) {
     public String getAccountId() {
         return ((UpholdAccountPayload) paymentAccountPayload).getAccountId();
     }
+
+    public String getAccountOwner() {
```

Comment:
Is there a reason why you don't display the account owner in `addFormForDisplayAccount`?

Reply:
Nope, my mistake :)